tests/test_data_compare_with_questions.py

```diff
@@ -0,0 +1,18 @@
+import io
+
```

Comment:
Do we need another test file now? I didn't find any reason to separate them.

I think pytest can help you create a BDD style test. I think your testing skill need some improvement. Read some books like THE ART OF THE UNIT TESTING or what

Reply:
I'm using doc-test to instead the test file here. Make it trackable and straightforward.